docs/index.md

```diff
@@ -2,51 +2,42 @@
 sidebar_position: 1
 ---
 
-# Introduction
+# Welcome to DBOS!
 
-Welcome to DBOS!
+With these DBOS offerings, you can easily build scalable, reliable, and secure TypeScript applications:
```

Comment:
I wouldn't say "offering" here. DBOS transact is and will always be open source. I think `With these DBOS solutions` would represent more accurately our commitment to OSS.